mate_bot/commands/pay.py

```diff
@@ -41,7 +41,10 @@ def __str__(self):
 class PayCommand(BaseCommand):
 
     def __init__(self):
-        super().__init__("pay", "")
+        super().__init__("pay", "The `/pay` command enables users who spend money for the community to get"
+                                " their money back. Using /pay <amount> <reason> you can claim you bought"
+                                " something for the community. Others have to approve before the money goes"
+                                " to your account.")
```

Comment:
1. The content is a bit short. Compare it to the description of `/communism`. Additionally, wait with the description of a command until it is implemented. As you can see, this is not the case for `/pay`.

2. We always put the space at the end of a line. Instead of

```python
" their money back. Using /pay <amount> <reason> you can claim you bought"
```

you should better use

```python
"their money back. Using /pay <amount> <reason> you can claim you bought "
```

3. Put the ` around command usages.